source/redux/send.js

```diff
@@ -78,11 +78,13 @@ export const sendToken = createAsyncThunk(
     const to = addressInfo.resolvedAddress || address;
     const amount = rawAmount.toString();
 
-    const { canisterId, standard, decimals } = selectedAsset || {};
-
+    const {
+      canisterId, standard, decimals, fee = 0,
+    } = selectedAsset || {};
+    const opts = { fee };
```

Comment:
dont you have to format the fee using the decimals?

Reply:
not here as the fee already comes formatted